net/packet/packet_test.go

```diff
@@ -74,18 +74,6 @@ var icmp4ReplyBuffer = []byte{
 	0x72, 0x65, 0x70, 0x6c, 0x79, 0x5f, 0x70, 0x61, 0x79, 0x6c, 0x6f, 0x61, 0x64,
 }
 
-var icmp4ReplyDecode = Parsed{
```

Comment:
Does this imply we are missing a test somewhere? (And likewise at 260)

Reply:
looks like the test exists, but uses a `Header.ToResponse()` method to convert the `icmp4RequestDecode` into the response header https://github.com/tailscale/tailscale/blob/cbeb8e7422fd61413cbab419e4453477b1bb7a38/net/packet/packet_test.go#L570-L572
same for udp4.